go.mod

```diff
@@ -329,3 +329,5 @@ require (
 	periph.io/x/conn/v3 v3.7.0 // indirect
 	periph.io/x/host/v3 v3.8.2 // indirect
 )
+
+replace go.viam.com/rdk => /Users/jeremyhyde/Development/rdk
```

Comment:
[note] to be removed after https://github.com/viamrobotics/rdk/pull/3376 and new rdk release